src/pages/en/guides/markdown-content.md

```diff
@@ -160,7 +160,7 @@ Astro.props = {
 
 #### Example: Using one Layout for `.md`, `.mdx`, and `.astro` files
 
-A single Astro layout can be written to receive the `frontmatter` object from `.md` and `.mdx` files, as well as any named props passed from `.astro` files.
+A single Astro layout can be written to receive the `frontmatter` object from `.md`, `.markdown` and `.mdx` files, as well as any named props passed from `.astro` files.
```

Comment:
```suggestion
A single Astro layout can be written to receive the `frontmatter` object from `.md` (or `.markdown` etc.) and `.mdx` files, as well as any named props passed from `.astro` files.
```
What do you think about changing the rest of the occurances to this, now that we've listed at the top of the `markdown-content` page??  @MoustaphaDev

Reply:
Perfect 👍️